### PR TITLE
Fix 87: Webpack (production) FavIcon, plus sorta-404 page

### LINKED ISSUE #87: Discrepancies Between Production/Staging Webpack?
There are a few issues with the production and staging versions of webpack. 

1) The favicon appears on staging, not on production
2) The AbeLincoln font is not imported correctly on production
3) Content/text is not pulled from the project resource on production

### PR DESCRIPTION
**Pull request by shaunanoordin**
## PR Overview
- This PR fixes #87 - or at least, it _should._ Testing the PR is a rather testing endeavour as I need to push things to production.
- Webpack now properly packs font files and the favicon.ico.
  - ONGOING: Still no idea why the Abe Lincoln font isn't being used correctly on production.
- ONGOING: investigating why project data isn't being pulled from Panoptes.

### Status
WIP, do not merge.

### CODE REVIEW

**Comment by shaunanoordin** [comment]
Update: the font issue can only be solved with this PR: https://github.com/zooniverse/static/pull/25

Please see the discussion on #87 for further details and an accurate stream of thought realisation of what the problem was.

**Comment by shaunanoordin** [comment]
Update: Project data visibility required two solutions:
1. The Project duck needs to be updated to not use the Temporary Hardcoded Project ID, which only works on staging.
2. In the PFE Project Builder, Project Visibility must be set from Private to Public.

**Comment by shaunanoordin** [comment]
## PR Update
- This PR now (correctly) chooses Project ID based on the environment, instead of using a hardcoded Project ID. Hello, Project Duck!
  - External requirement (DONE): PFE project 4973 has had its **Visibility set to Public.** (It's still in Development, not Live, mode.)
- External requirement: PR zooniverse/static#25 needs to be merged & deployed before the Abraham Lincoln font can be served properly on antislaverymanuscripts.org

### Status
Ready for review!

**Comment by shaunanoordin** [comment]
## Addendum
Forgot to mention two more things.

- If you try to access an invalid page, you actually see the website instead of just a blank white page. This helped with the debugging of the production stuff greatly, but that new `*` route in `index.js` is a good candidate for a 404 page.
- I've created a new Panoptes App and added its corresponding ID to the project. This _should_ allow sign in on production.

**Comment by wgranger** [comment]
Ahh. The visibility! Of course!

File: src/ducks/project.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ const classifierReducer = (state = initialState, action) => {
 
 //Action Creators
 
-const fetchProject = (id = TEMPORARY_HARDCODED_PROJECT_ID) => {
+const fetchProject = (id = config.zooniverseLinks.projectId) => {
```
Comment (wgranger) [inline src/ducks/project.js]: Ahh, nice. I made a similar change in https://github.com/zooniverse/liberating-the-liberator/pull/88, but I'll remove that fix in lieu of this after the merge.

**Comment by wgranger** [comment]
LGTM

**Pull request merged**